book/src/SUMMARY.md

```diff
@@ -0,0 +1,15 @@
+# Summary
+
+[Overview](overview.md)
```

Comment:
> instead of committing empty placeholders, please actually move the content from the README into the various book pages

Content will be landing once we iron out some build issues - mostly related to DRYing out the documentation.  
Which is why we kept the initial structure simple.  

If you intend to reject for incorrect structure please provide the structure/skeleton you want, either in the rFC or in a follow up PR to this one.

Given comments on prior PR's changes will continue to be as incremental as we can make them - if instead you want one large PR please say now.

Reply:
> Let's instead use the structure that already exists in README.md,

Will do.

Reply:
See above: this is not rejection, this is direction: I'd like to see the bulk of the content from `README.md` moved into this book, simply copying the structure that already exists in the README. We can iterate on it once that's done, but that's quick/easy.

So for example, a section titled [Getting Started](https://github.com/tag1consulting/goose/blob/main/README.md#getting-started), a section titled [Tips](https://github.com/tag1consulting/goose/blob/main/README.md#tips), a section titled [Simple Example](https://github.com/tag1consulting/goose/blob/main/README.md#simple-example), a section titled [Scheduling GooseTaskSets](https://github.com/tag1consulting/goose/blob/main/README.md#scheduling-goosetasksets) and so on.

I understand that this may not result in a perfectly logical book structure, but for me it's the first logical step and gives me something to review. We can then move things around into a more intuitive order and structure as followup: but I see this as inline with "Restructure (not rewrite) the documentation".